lib/live_view_demo/white_list.ex

```diff
@@ -0,0 +1,31 @@
+defmodule LiveViewDemo.WhiteList do
```

Comment:
Great idea to extract this to its own module 👏 👏 👏

Reply:
Just a comment.. we probably want to organize all sandbox modules under `LiveViewDemo.Sandbox` namespace. We can do it now or later in other PR. This would be `LiveViewDemo.Sandbox.WhiteList`